src/DatasetHandler.js

```diff
@@ -705,6 +705,7 @@ module.exports = class DatasetHandler {
     result.sourceRdfResources = `Vocabulary built from ${describeInput(
       this.vocabData,
     )}.`;
+    result.sourceTemplateVocab = describeTemplateVocab(this.vocabData);
```

Comment:
Ideally, separate features are added through separate PRs.

Reply:
Yeah, I know, but I just don't think we have the resource capacity to review multiple PRs in a timely fashion right now (unfortunately), as each review requires a costly 'context switch' for the reviewer.